README.md

```diff
@@ -119,29 +120,11 @@ To generate the Swift files, navigate to the `ldk-swift` repository and run the
 
 ```shell
 export LDK_SWIFT_GENERATOR_INPUT_HEADER_PATH="/path/to/ldk-c-bindings/lightning-c-bindings/include/lightning.h"
-python3 ./
+npm i && npm run tsc && node ./src/index.mjs
 ```
 
 Now, the contents of `./ci/LDKSwift/Sources/LDKSwift/bindings` will have been completely regenerated.
 
-### Preparing the correct Xcode version
```

Comment:
I removed this section since it seems to build fine with current Xcode 14.2. Let me know if that makes sense or if we should replace it with some less restrictive version requirements for Xcode.